package.json

```diff
@@ -4,11 +4,12 @@
   "description": "Type definitions for Corvid by Wix",
   "scripts": {
     "lint": "eslint .",
-    "build": "npm run generate-dts && npm run generate-full-corvid-types && webpack",
+    "build": "npm run generate-dts && npm run generate-full-corvid-types && npm run generate-module-list && webpack",
     "test": "jest",
     "test:debug": "node --inspect node_modules/.bin/jest",
     "generate-dts": "scripts/generate-types.js",
-    "generate-full-corvid-types": "scripts/generate-full-corvid-types.js"
+    "generate-full-corvid-types": "scripts/generate-full-corvid-types.js",
+    "generate-module-list": "scripts/generate-wix-module-list.js"
   },
   "repository": {
     "type": "git",
```

Comment:
i think you need to add wixModules.json to the files array in the package.json

Reply:
@matanlb - indeed, if you don't do that this file will not be included when installing the package

Reply:
True that, done